src/app/components/phone-field/index.html

```diff
@@ -6,7 +6,7 @@
 
   <sky-demo-page-summary>
     <p>
-      The phone field module contains a directive which validates phone numbers against the standards for different countries. The value that is validated against is driven through the <stache-code>ngModel</stache-code> attribute specified on the <stache-code>input</stache-code> element or a <stache-code>FormControl</stache-code> within a reactive form.
+      The phone field module creates a field and dropdown to validate international phone numbers. The phone field input directive validates the phone number that users enter in the field against standards for the country that they select in the dropdown. For validation, phone numbers are driven through the <stache-code>ngModel</stache-code> attribute that you specify on an <stache-code>input</stache-code> element or a <stache-code>FormControl</stache-code> in a reactive form.
```

Comment:
Same concerns on the `dropdown` language as that is changing.

Reply:
I'm reworking this paragraph.